src/ft.md

```diff
@@ -32,4 +32,43 @@ Awesome FTs Projects and Toolkits built with NEAR
 
 ## Token List
 
-- https://tkn.farm/
\ No newline at end of file
+- https://tkn.farm/
+
+## Token List Address(Testnet)
```

Comment:
```suggestion
## Frequently Used Tokens
```

Reply:
It would be nice to add some description under the section to describe why we add the frequently used tokens, and how do we collect and validate the addresses.